app/src/main/java/se/studieresan/studs/events/views/EventsFragment.kt

```diff
@@ -33,13 +34,12 @@ class EventsFragment : Fragment() {
     override fun onActivityCreated(savedInstanceState: Bundle?) {
         super.onActivityCreated(savedInstanceState)
         StudsApplication.applicationComponent.inject(this)
-        adapter = EventAdapter(requireActivity().applicationContext)
+        adapter = EventAdapter(requireActivity().applicationContext) { event -> displayEventDetails(event) }
         fetchEvents()
         swipe_refresh.setOnRefreshListener { fetchEvents() }
         rv_events.run {
             layoutManager = LinearLayoutManager(requireContext())
             adapter = this@EventsFragment.adapter
-            setHasFixedSize(true)
```

Comment:
Was this needed to be removed? Seems like there could be some performance gains by not removing it, but perhaps it needs to be removed by some other reason. 

Teach me like I'm five :heart_eyes:

Reply:
Nah, just tried different stuff. Added it back.